solr/core/src/java/org/apache/solr/core/SolrConfig.java

```diff
@@ -811,10 +818,18 @@ private void initLibs(SolrResourceLoader loader, boolean isConfigsetTrusted) {
     loader.reloadLuceneSPI();
   }
 
-  private void validateMemoryBreakerThreshold() {
+  private void validateCircuitBreakerThresholds() {
     if (useCircuitBreakers) {
-      if (memoryCircuitBreakerThresholdPct > 95 || memoryCircuitBreakerThresholdPct < 50) {
-        throw new IllegalArgumentException("Valid value range of memoryCircuitBreakerThresholdPct is 50 -  95");
+      if (isMemoryCircuitBreakerEnabled) {
+        if (memoryCircuitBreakerThresholdPct > 95 || memoryCircuitBreakerThresholdPct < 50) {
+          throw new IllegalArgumentException("Valid value range of memoryCircuitBreakerThresholdPct is 50 -  95");
+        }
+      }
+
+      if (isCpuCircuitBreakerEnabled) {
+        if (cpuCircuitBreakerThresholdPct > 95 || cpuCircuitBreakerThresholdPct < 40) {
+          throw new IllegalArgumentException("Valid value range for cpuCircuitBreakerThresholdPct is 40 - 95");
```

Comment:
I don't think CPU load average is typically measured on a 0-100 scale. Can you confirm some sample values of what calculateLiveCPUUsage returns?

Reply:
I see values between 0 - 100. Ran stress locally and validated values.

Reply:
Scratch that, this is a value that can exceed 100 as well. Removed the top ceiling on the value here.

Reply:
It typically doesn't go past 450 unless you have an issue. I would still keep it bounded at some measure @atris